City-of-Helsinki/hel-fi-drupal-grants
# Module & Theme updates February 9th 2022

Changes in this:

* Removed a development version of Entity Print (unused) from composer.json
* Updated helfi-modules
* Removed an unused admin theme (decision was made to use the default admin theme on platform level)
* ran drush cex to update the theme removal, which moved a line in a few hundred conf files around

To test:
* Install fresh, see that everything works
* check through the relevant files (use a filter to clear the .yml files to see general file changes, then at least check core.extension.yml that it's ok)
